core/src/Revolution/modActionDom.php

```diff
@@ -32,10 +32,10 @@ class modActionDom extends modAccessibleSimpleObject
      */
     public function apply($objId = '')
     {
+        /*
```

Comment:
Removing commented-out code would be preferred here.

Reply:
Can't believe I forgot to delete that! Doing so now...